web/source/osk/activeLayout.ts

```diff
@@ -209,11 +214,11 @@ namespace com.keyman.osk {
 
       // Add default right margin
       if(formFactor == 'desktop') {
-        totalWidth += 5;  // KMEW-117
+        totalWidth += ActiveKey.DEFAULT_RIGHT_MARGIN;  // KMEW-117
```

Comment:
This is probably wrong; I have not really tested it -- but it's different to what it was previously.

Reply:
I've gone ahead and restored the original value for desktop (now we can see where the 5 came from in Keyman Developer's builder.js, so that this PR doesn't impact layout. We can sort that one value out in the future.